src/library_wasmfs.js

```diff
@@ -286,6 +286,12 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       return FS.handleError(__wasmfs_ftruncate(fd, {{{ splitI64('len') }}}));
     },
     // TODO: utime
+    utime: (path, atime, mtime) => {
+      return FS.handleError(withStackSave(() => {
```

Comment:
You can remove the `return` here and the curly brace before it.  Similar to #19614